tests/bluetooth/buf/testcase.yaml

```diff
@@ -0,0 +1,20 @@
+common:
+  tags:
+    - bluetooth
+    - host
+
+tests:
+  bluetooth.buf:
+    platform_allow:
+      - native_sim
+      - native_sim/native/64
+    integration_platforms:
+      - native_sim
```

Comment:
Should we add extra_configs here as well to make sure the config is as we expect, no matter what the default is?

Reply:
Right, though if we check the exact returned value, this may not be needed. I added `CONFIG_BT_HCI_ACL_FLOW_CONTROL=y` any way.